spec/inputs/dynamodb_spec.rb

```diff
@@ -0,0 +1,207 @@
+# encoding: utf-8
+#
+#Copyright 2015 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+#Licensed under the Apache License, Version 2.0 (the "License");
+#you may not use this file except in compliance with the License.
+#You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+#Unless required by applicable law or agreed to in writing, software
+#distributed under the License is distributed on an "AS IS" BASIS,
+#WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#See the License for the specific language governing permissions and
+#limitations under the License.
+#
+require "logstash/devutils/rspec/spec_helper"
+require "logstash/inputs/dynamodb"
+require "rspec/expectations"
+require "rspec/mocks"
+require "mocha"
+
+RSpec.configure do |config|
+  config.mock_with :mocha
+end
+
+class LogStash::Inputs::TestDynamoDB < LogStash::Inputs::DynamoDB
```

Comment:
I would recommend moving this mocks and configurations to their own ```spec_helper.rb``` file. You can do something like moving this content so the a new file into ```spec/spec_helper.rb```, including the ```require "logstash/devutils/rspec/spec_helper"``` and then to a ```require_relative '.../spec_helper.rb'```  from this test.